spec/facility_spec.rb

```diff
@@ -22,5 +27,82 @@
+      expect(@facility_1.administer_written_test(registrant_1)).to eq(false)
+      @facility_1.add_service('Written Test')
+      @facility_1.administer_written_test(registrant_1)
+      expect(@facility_1.administer_road_test(registrant_1)).to eq(false)
+    end
+  end
+  describe '#register vehicle' do
+    it 'can register a vehicle' do
+      @facility_1.add_service('Vehicle Registration')
+      @facility_1.register_vehicle(@cruz)
+      @facility_1.register_vehicle(@bolt)
+      @facility_1.register_vehicle(@camaro)
+      expect(@cruz.plate_type).to eq(:regular)
+      expect(@bolt.plate_type).to eq(:ev)
+      expect(@camaro.plate_type).to eq(:antique)
+      expect(@cruz.registration_date).to eq(Date.today)
+      expect(@bolt.registration_date).to eq(Date.today)
+      expect(@facility_1.collected_fees).to eq(325)
+      expect(@facility_1.registered_vehicles).to eq([@cruz, @bolt, @camaro])
+      expect(@facility_2.registered_vehicles).to eq([])
+      expect(@facility_2.register_vehicle(@bolt)).to eq(nil)
+    end
+  end
+  describe '#administer_written_test' do
+    it 'can give written tests' do
+      registrant_1 = Registrant.new('Bruce', 18, true )
+      @facility_1.add_service('Written Test')
+      expect(@facility_1.services).to eq(["Written Test"])
+      expect(@facility_1.administer_written_test(registrant_1)).to eq(true)
+      expect(registrant_1.license_data).to eq({:written=>true, :license=>false, :renewed=>false})
+    end
+    it 'can check attributes required for test taking' do
+        registrant_2 = Registrant.new('Penny', 16 )
+        registrant_3 = Registrant.new('Tucker', 15 )
+        @facility_1.add_service('Written Test')
+        expect(@facility_1.administer_written_test(registrant_2)).to eq(false)
+        expect(@facility_1.administer_written_test(registrant_3)).to eq(false)
+        registrant_2.earn_permit
+        expect(@facility_1.administer_written_test(registrant_2)).to eq(true)
+        registrant_3.earn_permit
+        expect(@facility_1.administer_written_test(registrant_3)).to eq(false)
+    end
+  end
+  describe '#administer_road_test' do
+    it 'can give road tests' do
+      registrant_1 = Registrant.new('Bruce', 18, true )
+      @facility_1.add_service('Written Test')
+      @facility_1.add_service('Road Test')
+      @facility_1.administer_written_test(registrant_1)
+      @facility_1.administer_road_test(registrant_1)
+      expect(@facility_1.services).to eq(["Written Test", "Road Test"])
+      expect(@facility_1.administer_road_test(registrant_1)).to eq(true)
+      expect(registrant_1.license_data).to eq({:written=>true, :license=>true, :renewed=>false})
+    end
+    it 'can check attributes required to take road test' do
+      registrant_3 = Registrant.new('Tucker', 15 )
+      @facility_1.add_service('Written Test')
+      @facility_1.add_service('Road Test')
+      expect(registrant_3.license_data).to eq({:written=>false, :license=>false, :renewed=>false})
+    end
```

Comment:
I'm not sure what this test is accomplishing. We haven't done anything so I would definitely expect this license_data to still be all false.